index.html

```diff
@@ -1,7 +1,51 @@
 <!doctype html>
+<html lang="en">
 <head>
-<title>HackYU2021</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.13/semantic.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.13/semantic.min.js"></script>
+    <title>HackYU2021</title>
 </head>
 <body>
-    <h1> HackYU2021 page! </h1>
+<header>
```

Comment:
Is this header going to be copied to the other pages? We should definitely have a header with home, schedule, faq, and sign-up links on all of the pages, but will we also have links to about, guests, sponsors, and contact info? Seeing as these last few are all components of the home page, I'm not sure if they should be part of the headers on other pages. (Maybe you disagree?) If you do agree that we shouldn't include them in the headers on other pages, it might make sense to put about, guests, sponsors, and contact info below home, schedule, faq, and sign-up links.

Reply:
I agree that we need to put them on all the pages, great catch! 
I put the parts of same home page (about, guests, sponsors, and contact) in the header with the other pages in such way that we can make the dropdown from home button. If we want to take them out in a different nav bar for example on the side, we can do that as well. Let me know because this formatting has just to do with the design choice.

Reply:
Ok, I understand - I didn't realize it was going to be a drop-down menu, I think that's a good idea. The only thing is that now that I realized it's a dropdown, maybe it would makes sense to copy the same exact header - meaning including the subnav - for the other pages (including the drop-down menu)? To me, I think it wouldn't make sense to have the header be different on different pages...

Reply:
The way I have seen it done in the past (which was the reason that I have done it this way and we can totally do it some different way) is that there is the "active" flag (the class) on the page we are currently on and only that has the dropdown. Meaning that one cannot access the dropdown of the home page from the schedule page, but they can access different parts of the schedule page, because it would have the flag active. The reason I have not added a dropdown menu on other pages is because I don't really know what are going to be the parts of other pages, they seem to be more monolithic and only related to one single topic. Let me know if what I am explaining makes any sense at all :D

Reply:
If you've seen it done that way, then I guess that's fine - you definitely have a lot more experience with this than I do!